src/formatter.rs

```diff
@@ -63,16 +63,17 @@ fn format_code(command: &str, text: &str, config: &Config) -> String {
 }
 
 /// Print a token stream to an ANSI terminal.
-pub fn print_lines<R>(tokenizer: &mut Tokenizer<R>, config: &Config)
```

Comment:
I think it's convention that the types are listed in the same order they are used, but I am not sure :thinking:

Reply:
I’ve never thought about it, happy to fix..

Reply:
I'm not aware of that convention, I usually try to put them in a logical order. Sometimetimes no logical order is obvious though 🙂